app/controllers/application_controller.rb

```diff
@@ -11,7 +11,9 @@ class ApplicationController < ActionController::Base
   fine_print_require :general_terms_of_use, :privacy_policy, unless: :disable_fine_print
 
   def redirect_to_newflow_if_enabled
-    if request.get? && Settings::Db.store.student_feature_flag && !params[:bpff].present?
+    return false unless request.get?
```

Comment:
if you're returning boolean, the ruby convention for this is to end the method in a ?

https://blog.codeminer42.com/and-understanding-one-of-rubys-coolest-naming-conventions-5a9300b75605/

Reply:
Changed to `return unless request.get?` since the return value doesn't matter.